extensions/interactions/MusicNotesInput/directives/music-notes-input-rules.service.ts

```diff
@@ -35,8 +38,9 @@ export class MusicNotesInputRulesService {
     if (
       InteractionsExtensionsConstants.NOTE_NAMES_TO_MIDI_VALUES.hasOwnProperty(
         note.readableNoteName)) {
+      const readableNoteName_ = note.readableNoteName as ReadableNoteNames;
```

Comment:
```suggestion
      const _readableNoteName: ReadableNoteNames = note.readableNoteName;
```
Using underscore before variable names looks more better or you can use a different name, Can you make the changes here and elsewhere in this PR ?

Reply:
done